src/state/type-filter-params-state.ts

```diff
@@ -0,0 +1,14 @@
+import { atom } from 'recoil';
+
+export interface TypeFilter {
+  filter: string;
+  active: boolean;
+}
+
+/*
+ * Recoil state to manage the Type Filter parameters.
+ */
+export const activeTypeFilterParamsState = atom<TypeFilter[]>({
```

Comment:
A bit of a nitpick, but only some of the filters that this state is used for are active. Do we want to just call this `typeFilterParamsState` instead?